BinaryObjectScanner/Protection/CopyX.cs

```diff
@@ -0,0 +1,182 @@
+        // either 0x00, or some data that matches between entries (and also is present in the 3 Professional files),
+        // except for the parts with the rings running through them.
+        // TODO: Check the last directory alphabetically and not just ZDAT*
+
+        /// <inheritdoc/>
+        public string? CheckPortableExecutable(string file, PortableExecutable pex, bool includeDebug)
+        {
+            // Checks for Professional
+            var sections = pex.Model.SectionTable;
+            if (sections == null)
+                return null;
+            if (pex.OverlayStrings != null)
+            {
+                // Checks if main executable contains reference to optgraph.dll. Emergency 4's is missing this for some reason. 
+                // This might be better removed later, as Redump ID 82475 is a false positive, and also doesn't actually
+                // contain the actual optgraph.dll file.
+                // TODO: Find a way to check for situations like Redump ID 48393, where the string is spaced out with
+                // 0x00 between letters and does not show up on string checks.
+                // TODO: This might need to check every single section. Unsure until more samples are acquired.
+                if (pex.OverlayStrings.Any(s => s.Contains("optgraph.dll")))
+                {
+                    // TODO: TKKG also has an NE 3.1x executable with a reference. This can be added later.
+                    // Samples: Redump ID 108150
+                    return "copy-X";
+                }
+            }
+
+            var strs = pex.GetFirstSectionStrings(".rdata");
+            if (strs != null)
+            {
+                // Samples: Redump ID 82475 (False positive, but probably catches original Emergency 2), would catch Redump ID 48393
+                if (strs.Any(s => s.Contains("optgraph.dll")))
+                    return "copy-X";
+            }
+
+            return null;
+        }
+
+        /// <inheritdoc/>
+#if NET20 || NET35
+        public Queue<string> CheckDirectoryPath(string path, IEnumerable<string>? files)
+#else
+        public ConcurrentQueue<string> CheckDirectoryPath(string path, IEnumerable<string>? files)
+#endif
+        {
+#if NET20 || NET35
+            var protections = new Queue<string>();
+#else
+            var protections = new ConcurrentQueue<string>();
+#endif
+
+            // Checks for Light
+            if (files == null)
+                return protections;
+
+            // Gets files in ZDAT*
+            // Excludes files with .x64 extension to avoid flagging Professional files.
+            var fileList = files.Where(f => !f.EndsWith(".x64", StringComparison.OrdinalIgnoreCase)).Where(f =>
+            {
+                f = f.Remove(0, path.Length);
+                f = f.TrimStart('/', '\\');
+                return f.StartsWith("ZDAT", StringComparison.OrdinalIgnoreCase);
+            }).OrderBy(f => f).ToList();
```

Comment:
Each Linq method should be on its own line. For example:
```c#
.OrderBy(f => f).ToList()
```
should become
```c#
.OrderBy(f => f)
.ToList()
```

Also please check indentation. This won't align at the same level that `var fileList` is at.